tests/conftest.py

```diff
@@ -26,8 +26,8 @@
 
 
 @pytest.fixture
-def tol(analytic):
-    if analytic:
+def tol(shots):
+    if not shots:
```

Comment:
Just so that we're sure it only every works for `None`
```suggestion
    if shots is None:
```